src/docs/background/privileged-transitions/index.md

```diff
@@ -0,0 +1,87 @@
+transitioned and by defining which party of a transaction can perform
+the transition. This way transition requests have built-in validation of
+who can invoke them and in what state of the transaction flow.
+
+However, there are moments when more control is required on who can
+initiate a transition and especially with what kind of parameters. Take
+discounts in pricing by leveraging discount coupons managed by a 3rd
+party service for example. A coupon code can be validated in the client
+side by invoking the coupon service but this will not limit who will be
+technically able to invoke a pricing related transition with discounted
+price parameters.
+
+This is where privileged transitions come into play. They are
+transaction process transitions that can be invoked from a secure
+context. In other words, this means that you can build your own server
+side validation that sits between your marketplace UI and the Flex
+Marketplace API. In the discount coupon example, this means that the
+discount coupon that a user has can be passed as a parameter in the
+transition request. Server side transition request validation can invoke
+a 3rd party service to verify a discount code, update pricing parameters
+accordingly, and pass those to the transition that has pricing actions
+tied to it.
+
+## How do privileged actions work?
+
+Standard authenticated Marketplace API requests require a valid access
+token obtained from the Authentication API. Privileged transitions
+differ here by requiring a special kind of _trusted_ token to
+authenticate properly. A trusted token can be obtained by exchanging a
+valid access token to a trusted one in the Authentication API by
+providing a client secret. The client secret is not to be exposed
+publicly, securing that privileged transitions can only be invoked from
+a trusted source, i.e. your own backend implementation.
+
+![Authentication flow with a trusted access token](auth-flow.png)
+
+The client secret is tied to a Marketplace API
+[application](/background/applications/). When exchanging an access
+token to a trusted one, the client secret needs to be from the same
+application as the client ID that was used to obtain the access token.
+
+Remember to never expose the client secret publicly. This enables full
+control over requests that invoke privileged transitions.
+
+## How to use a privileged transition
+
+A privileged transition is defined by setting a `privileged?` attribute
+to `true` for the given transition in a transaction process as follows:
+
+```
+{:name :transition/request-payment
+ :actor :actor.role/customer
+ :actions [{:name :action/create-booking
+            :config {:observe-availability? true}}
+           {:name :action/privileged-set-line-items}
+           {:name :action/stripe-create-payment-intent}]
+ :to :state/pending-payment
+ :privileged? true}
+```
+
+A privileged transition works in combination with privileged actions.
```

Comment:
This is a great chapter, but I would like to see an addition that the only requirement for privileged transition is that you add the `privileged?` attribute and can attach any actions to it. The limitation is for actions requiring this flag to be on, but in theory you could use this for updating protected data as well (that's one of the obvious ones where heightened security would be nice), or any other action.